kitchen.global.yml

```diff
@@ -3,9 +3,6 @@ provisioner:
   attributes:
     cluster:
       # right now tests depend on this parameter: we will try to remove the dependency later
-      base_os: alinux
+      base_os: alinux2
       region: us-east-1
-      dcv:
-        server: {}
-      parallelcluster-node-version: '3.4.1'
-      node_type: 'HeadNode'
+      parallelcluster-node-version: '3.4.1'
```

Comment:
Do we still need this?

Reply:
This attribute is needed to be able to install an already published parallelcluster-node-version

Reply:
Do we really want to install `3.4.1`?

Reply:
It is ok since it is not used nor checked. Any released version is ok.